src/Utilities/UnitTests/ToolTask_Tests.cs

```diff
@@ -31,7 +31,7 @@ public MyTool()
 #else
                     FileUtilities.GetFolderPath(FileUtilities.SpecialFolder.System),
 #endif
-                    NativeMethodsShared.IsUnixLike ? "/bin/sh" : "cmd.exe");
+                    NativeMethodsShared.IsUnixLike ? "sh" : "cmd.exe");
```

Comment:
Can you add a note in the commit message about why you're switching away from `/bin/`?